documentation/markup.rst

```diff
@@ -22,7 +22,7 @@ Element                 Markup                                      See also
 arguments/parameters    ``*arg*``                                   :ref:`inline-markup`
 variables/literals/code ````foo````, ````42````, ````len(s) - 1```` :ref:`inline-markup`
 True/False/None         ````True````, ````False````, ````None````   :ref:`inline-markup`
-function definitions   ``.. function:: print(*args)``               :ref:`directives`
+function definitions   ``.. function:: print(*args)``              :ref:`directives`
 function references    ``:func:`print```                            :ref:`roles`
 attribute definitions   ``.. attribute: `attr-name```               :ref:`information-units`
```

Comment:
Should this look like this?
```suggestion
function definitions    ``.. function:: print(*args)``              :ref:`directives`
function references     ``:func:`print```                           :ref:`roles`
attribute definitions   ``.. attribute: `attr-name```               :ref:`information-units`
```